.gitignore

```diff
@@ -1,6 +1,6 @@
 /build/cache/*
 pkg/assets/zz_generated_offsets*.go
-k0s
+./k0s
```

Comment:
Sure this doesn't inflict any harm on the docs build? IIRC It's downloading the k0s binary into the docs folder.

Reply:
This should be `/k0s` to exclude the built `k0s` binary. There are others that can be scoped to the root of the project, but this one alone is easily the most needed change.

I just hit this problem again as autopilot has paths with `/k0s/` in them, resulting in them being incorrectly ignored.

Reply:
Okay, I suggest to change this as follows, then:

```diff
diff --git a/.gitignore b/.gitignore
index c54ccd97a..a9fd75e57 100644
--- a/.gitignore
+++ b/.gitignore
@@ -1,9 +1,9 @@
 /build/cache/*
-pkg/assets/zz_generated_offsets*.go
-./k0s
-k0s.code
-k0s.exe
-k0s.exe.code
+/k0s
+/k0s.code
+/k0s.exe
+/k0s.exe.code
+/pkg/assets/zz_generated_offsets*.go
 bindata*
 .*.stamp
 .tmp
diff --git a/docs/.gitignore b/docs/.gitignore
index e71dd39e6..4f2b5aedd 100644
--- a/docs/.gitignore
+++ b/docs/.gitignore
@@ -1,2 +1,4 @@
 /*.etag
 /.docker-image.*
+/k0s
+/k0s.exe
```

Reply:
Done